cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -102,10 +102,7 @@ spec:
         hostPath:
           path: /lib/modules
       tolerations:
-      - key: CriticalAddonsOnly
-        operator: Exists
-      - key: {{ .MasterTaintKey }}
-        effect: NoSchedule
+      - operator: Exists
```

Comment:
You need to add back the CriticalAddons toleration b/c it's used by the kubelet and the scheduler - https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/